src/app/campaign/flight/flight-form-control-container.component.ts

```diff
@@ -89,7 +89,7 @@ export class FlightFormControlContainerComponent implements OnInit, OnDestroy {
     id: [],
     name: ['', Validators.required],
     startAt: ['', Validators.required],
-    endAt: ['', Validators.required],
+    endAtFudged: ['', Validators.required],
     contractStartAt: [''],
     contractEndAt: [''],
```

Comment:
Hrmm... I wonder if we should do the same thing with the contract end date.  They'll be setting it using the same "inclusive end date" logic.